config/prow/jenkins/jenkins-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: jenkins
```

Comment:
@sudeeshjohn Can we please move this namespace resource to https://github.ibm.com/powercloud-secrets/secrets/blob/master/projects/k8s-prow/0001-namespaces.yaml?

Can be part of this PR https://github.ibm.com/powercloud-secrets/secrets/pull/29.

Reply:
moved it to secret repo